v2/src/ctrlf_auth/views.py

```diff
@@ -53,3 +54,16 @@ def post(self, request, *args, **kwargs):
             for _, message in serializer.errors.items():
                 message = message[0]
         return Response(data={"message": message}, status=status.HTTP_400_BAD_REQUEST)
+
+
+class CheckEmailDuplicateView(APIView):
+    @swagger_auto_schema(query_serializer=CheckEmailDuplicateSerializer)
+    def get(self, request):
+        serializer = CheckEmailDuplicateSerializer(data=request.GET)
```

Comment:
`request.GET` 보단 `request.query_params` 로~


FYI, https://www.django-rest-framework.org/api-guide/requests/#query_params
```
request.query_params is a more correctly named synonym for request.GET.

For clarity inside your code, we recommend using request.query_params instead of the Django's standard request.GET.
```